LICENSE

```diff
@@ -1,8 +1,8 @@
-Unless it has its own copyright/license embedded in its body, each source file
-is subject to the following license terms:
+This software contains materials under the terms of the BSD-2-Clause license
+and materials under the terms of the MIT license as stated in the related
+files.
```

Comment:
"as stated in the related files" is ambiguous, or even incorrect, because libyaml files contain no licensing terms (although libyaml/ has a LICENSE file).

IMO we should be more explicit:

```
1. Files under libyaml/ are covered by the following license:

<Insert libyaml/LICENSE here>

2. All other files are covered by the following license (BSD-2-Clause):

<Insert BSD-2-Clause text here>
```

Reply:
ok, will do. thanks.